recipes/kipoi/meta.yaml

```diff
@@ -1,29 +1,27 @@
 {% set name = "kipoi" %}
-{% set version = "0.3.6" %}
+{% set version = "0.5.5" %}
 {% set file_ext = "tar.gz" %}
 {% set hash_type = "sha256" %}
-{% set hash_value = "6a2482e91c4fd168411bfb5c8c5ad7833669f6c02ad2af8f3ec21cac06541302" %}
+{% set hash_value = "e47e71aabcac7cd2d15bd570be6c0b86203bf66a4594a59217ddfa5299af2026" %}
 
 package:
   name: '{{ name|lower }}'
   version: '{{ version }}'
 
 source:
-  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
   '{{ hash_type }}': '{{ hash_value }}'
 
 build:
-  number: 1
+  number: 0
   entry_points:
     - kipoi = kipoi.__main__:main
   script: pip install 'related>=0.6.0' enum34 epc && python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
This sounds rather wrong. Why are enum32, rpc and related pip installed rather than pulled in as dependencies?

Reply:
ping @visze @Avsecz